lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -533,7 +533,7 @@ def add_resources_bundle_targets(file_accessors)
                   device_family_by_platform = {
                     :ios => '1,2',
                     :tvos => '3',
-                    :watchos => '1,2' # The device family for watchOS is 4, but Xcode creates watchkit-compatible bundles as 1,2
+                    :watchos => '1,2,4' # The device family for watchOS is 4, but Xcode creates watchkit-compatible bundles as 1,2. Adding wathOS device family to resolve warning.
```

Comment:
this comment is a bit unnecessary both the old one and the added one you had. Can we remove entirely?